### PR TITLE
Add eip155-13 (Dehvo)

### PR DESCRIPTION
**Pull request by Phinetwork**
UPDATED ADD CHAIN ID 13 OFFICIAL CHAIN DEHVO

### CODE REVIEW

**Review by kophyo1234 — APPROVED** [review]

**Comment by Phinetwork** [comment]
@ligi can we go ahead and merge this pull request thank you 🙏

**Comment by Phinetwork** [comment]
Can we merge this commit @kophyo1234 @ligi @

**Comment by Phinetwork** [comment]
Hello @ligi

**Comment by Phinetwork** [comment]
Hello can we please get this merged

File: _data/chains/eip155-13.json [hunk 1]
```diff
@@ -1,18 +1,25 @@
 {
-  "name": "Diode Testnet Staging",
-  "chain": "DIODE",
+  "name": "Dehvo",
+  "chain": "Dehvo",
+  "network": "mainnet",
   "rpc": [
-    "https://staging.diode.io:8443/",
-    "wss://staging.diode.io:8443/ws"
+    "https://connect.dehvo.com",
```
Comment (ligi) [inline _data/chains/eip155-13.json:6]:
```suggestion
    "https://connect.dehvo.com"
```

**Comment by Phinetwork** [comment]
can we merge this pull request thank you

**Comment by Phinetwork** [comment]
> can we merge this pull request thank you 

@kophyo1234

**Comment by Phinetwork** [comment]
@ligi He @kophyo1234 hasnt been in over a month can we go ahead and merge this idk what he is doing but we dont have months to wait for him

**Comment by Phinetwork** [comment]
This is ready to merge

**Comment by ligi** [comment]
can you elaborate on this change?

**Comment by Phinetwork** [comment]
The chain ID currently for 13 is a deprecated chain and not currently live chain ID 13 is dehvo this needs to be added dehvo chain is chain id 13 thank you

**Comment by ligi** [comment]
The other one is still active

**Comment by ligi** [comment]
image

looks active to me

**Comment by Phinetwork** [comment]
> image
> 
> 
> 
> looks active to me

It's not chain ID 13 because chain ID 13 is https://explorer.dehvo.com

**Comment by ligi** [comment]
?

**Pull request closed without merging**